### ISSUE TITLE
Page RE: How neural networks learn

### ISSUE THREAD

**Issue by tedmoore** [flucoma/learn-website#59]
With a description of things like:

* how to interpret the "loss" value
* randomized state
* parameters vs. internal parameters (...and/or... hyperparameters vs. parameters)
* activations
* layers
* neurons
* from @weefuzzy (https://github.com/flucoma/learn-website/pull/54) re: learn rate "The most important hyper-parameter. Crucial to find the operative range for a given combination of data and NN as part of the iterative process of training. Start with course, order-of-magnitude adjustments and find the lower limit where it doesn't converge, and the upper limit where it starts to get so noisy it can diverge. The goldilocks point is often just a bit shy of this upper limit."
* ditto: re: batch size: "Affects the speed and even-ness of convergence (too small, quick but noisy adjustment; too large, smooth but very slow)"

**Comment by tedmoore** [comment]
@jamesb93 or an overview, or a block of text that appears in both MLPs.
